doc/releases/changelog-dev.md

```diff
@@ -49,6 +49,22 @@
 
 <h3>Deprecations 👋</h3>
 
+* The ``expand_fn`` argument in ``qml.execute`` has been deprecated. 
```

Comment:
No need for double backticks in the changelog as it is markdown. The double back ticks are for docstrings and `.rst` files. Can you fix it for the rest of the changelog as well?
```suggestion
* The `expand_fn` argument in `qml.execute` has been deprecated. 
```

Reply:
Sounds good, thank you!